### PR TITLE
KOGITO-7260: Patch PR for KOGITO-7250 with doc improvements in input schema guide

### PR DESCRIPTION
**Pull request by hmanwani-rh**
**JIRA:** https://issues.redhat.com/browse/KOGITO-7260

**Description:** Patch PR for KOGITO-7250 with doc improvements in input schema guide

https://github.com/kiegroup/kogito-docs/pull/151

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to setup automatic cherry-pick to another branch ?
</summary>

The cherry-pick action allows to setup automatic cherry-pick from `main` to a specific branch.

To allow it, you will need to add the corresponding label with pattern: `backport-{RELEASE_BRANCH}`.  
For example, if a backport to branch `1.26.x` is needed, then the label should be `backport-1.26.x`.

Once the PR is merged, the action will retrieve the commit and cherry-pick it to the desired branch.

*NOTE: You can still add label after the merge and it should still be cherry-picked.*
</details>

### CODE REVIEW

**Comment by hmanwani-rh** [comment]
@VaniHaripriya Can you please review the changes in this PR and share your comments?

File: serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc [hunk 1]
```diff
@@ -43,4 +39,5 @@ components:
         message:
           type: string
 ----
-[source,yaml]
+
+include::../../pages/_common-content/report-issue.adoc[]
```
Comment (ricardozanini) [inline serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc:43]: @hmanwani-rh I forgot to add, but what do you think about adding additional resources here to give more context?

```asciidoc
== Additional resources

* xref:service-orchestration/orchestration-of-openapi-based-services.adoc[Orchestrating the OpenAPI services]
* xref:service-orchestration/configuring-openapi-services-endpoints.adoc[Configuring the OpenAPI services endpoints]
```

Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc:43]: sure, we can.

Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc:48]: Done, added Additional resources.

**Review by VaniHaripriya — APPROVED** [review]
@hmanwani-rh The changes looks good to me Heena. Thank you!

**Pull request merged**